src/tools/base/BaseTool.js

```diff
@@ -1,4 +1,5 @@
 import mixins from './../../mixins/index.js';
+import lodash from 'lodash';
```

Comment:
Is there any way we can depend on _only_ `lodash`'s `merge` method instead of all of `lodash`?

Reply:
Bonus points if we can pull in just this utility method's source to reduce the number of external dependencies we have.

Reply:
Good point indeed! But I thought about that too. 

We have lodash to perform a merge function for webpack config. 
We could easily change from the whole lodash to only cloneDeep, merge and mergeWith. 
But some of our npm packages also uses the whole lodash module, so dividing it to lodash.clonedeep/merge/mergewith will actually increase the size of our package.. at least the map increased in my tests.

Reply:
Fixed this issue.